api/KeyboardEvent.json

```diff
@@ -177,10 +177,10 @@
               "version_added": "12.1"
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "1.2"
```

Comment:
`charCode` was on `UIEvent` in https://trac.webkit.org/browser/webkit/branches/old/Safari-1-2-branch/WebCore/khtml/dom/dom2_events.h so let's treat it as 1.2.